.github/workflows/test-library.yml

```diff
@@ -356,4 +561,97 @@ jobs:
       run: >-
         print("All's good")
       shell: python
+
+  publish-pypi:
+    name: Publish 🐍📦 ${{ needs.pre-setup.outputs.git-tag }} to PyPI
+    needs:
+    - check
+    - pre-setup  # transitive, for accessing settings
+    if: >-
+      fromJSON(needs.pre-setup.outputs.release-requested)
+    runs-on: Ubuntu-latest
+
+    environment:
+      name: release
```

Comment:
So, IIUC, we can also deprecate the `master` branch workflow after this?  Because, we will be able to add tags and push to PyPi or TestPyPi directly from the `develop` branch.

Reply:
> So, IIUC, we can also deprecate the `master` branch workflow after this?  Because, we will be able to add tags and push to PyPi or TestPyPi directly from the `develop` branch.

You could, if you want. Or you could still maintain an extra branch. It isn't directly related to the tags, strictly speaking.